ui/app/unlock.js

```diff
@@ -70,7 +70,7 @@ UnlockScreen.prototype.render = function () {
 
       h('.flex-row.flex-center.flex-grow', [
         h('p.pointer', {
-          onClick: () => this.props.dispatch(actions.showRestoreVault()),
+          onClick: () => this.props.dispatch(actions.forgotPassword()),
```

Comment:
An easier way to have done this whole commit would have been to add a line of `this.props.dispatch(actions.transitionForwards())` before `showRestoreVault`.

That said, you did this fast, thank you!